pkg/apis/config/v1alpha1/resmgr/policy/balloons/config.go

```diff
@@ -220,11 +221,13 @@ type BalloonDef struct {
 	ShareIdleCpusInSame CPUTopologyLevel `json:"shareIdleCPUsInSame,omitempty"`
 	// PreferCloseToDevices: prefer creating new balloons of this
 	// type close to listed devices.
+	// +listType=set
```

Comment:
The documentation says:

"Equality on arrays with x-kubernetes-list-type of set or map ignores element order."

Would this mean that if a user changes the order of items in the PreferCloseToDevices list, then some validation logic could say that a CR has not been changed? Or that it does not matter in which order items in the list are displayed or stored? Or that CEL expressions that would compare these lists of two balloon types would consider preferences similar only if they contain the same devices?

Any of these cases would be problematic, because we document that the order of items in this list is important. A device listed before another device has higher preference than the other.